### PR TITLE
PMM-6433 check kubernetes connection

### PR DESCRIPTION
**Pull request by BupycHuk**
[PMM-6433](https://jira.percona.com/browse/PMM-6433)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1135

- [x] Links to other linked pull requests (optional).
https://github.com/percona-platform/dbaas-controller/pull/60  
https://github.com/Percona-Lab/pmm-submodules/pull/1135  
https://github.com/percona-platform/dbaas-api/pull/28  
https://github.com/Percona-Lab/pmm-api-tests/pull/128  
---
- [x] Tests passed.
- [ ] Feature build pass.
- [x] (Re)requested review.
- [x] Fix conflicts with target branch.
- [x] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

**Review by artemgavrilov — APPROVED** [review]

**Review by askomorokhov — APPROVED** [review]

**Comment by AlekSi** [comment]
There are conflicts now

**Comment by BupycHuk** [comment]
Will fix

**Comment by AlekSi** [comment]
Please merge with PMM-2.0 again

File: Gopkg.toml [hunk 1]
```diff
@@ -16,12 +16,16 @@ required = [
...
-  branch = "PMM-2.0"
+  branch = "PMM-6457-api-db-cluster"
 
 [[constraint]]
   name = "github.com/percona-platform/saas"
   branch = "main"
 
+[[constraint]]
+  name = "github.com/percona-platform/dbaas-api"
+  branch = "PMM-6433-check-kubernetes-connection"
```
Comment (AlekSi) [inline Gopkg.toml:27]: Let's keep that constraint with `branch = "main"` after that PR

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
More conflicts

**Pull request merged**